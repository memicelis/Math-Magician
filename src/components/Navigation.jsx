import React from 'react';
import { Link } from 'react-router-dom';
import './Navigation.module.css';

const Navigation = () => (
  <>
    <nav>
      <h1>Math Magician</h1>
      <ul>
        <li>
          <Link to="/">Home</Link>
        </li>
        <li>
          <Link to="/calc">Calculator</Link>
        </li>
        <li>
          <Link to="/quote">Quote</Link>
        </li>
      </ul>
    </nav>
  </>
);

export default Navigation;
