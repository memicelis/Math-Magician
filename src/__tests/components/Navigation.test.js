import React from 'react';
import Navigation from '../../components/Navigation';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

it('testing Navigation component', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <Navigation />
      </MemoryRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
