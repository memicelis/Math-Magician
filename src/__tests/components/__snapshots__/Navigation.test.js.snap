// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Navigation component 1`] = `
<nav>
  <h1>
    Math Magician
  </h1>
  <ul>
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/calc"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
